protocol/messenger.go

```diff
@@ -2846,7 +2846,7 @@ func (m *Messenger) SyncDevices(ctx context.Context, ensName, photoPath string)
 			}
 		}
 
-		if (isPublicChat || chat.OneToOne() || chat.PrivateGroupChat()) && !chat.Active {
+		if (isPublicChat || chat.OneToOne() || chat.PrivateGroupChat()) && !chat.Active && chat.DeletedAtClockValue > 0 {
```

Comment:
@cammellos I've extended the condition here instead of replacing it (`!Active && DeletedAtClockValye > 0`) to be sure.
Also, the `Clock` used in `SyncChatRemoved` message, I've reverted to being the `clock` it was before (instead of the `DeletedAtClockValue`).

Reason being, one of the tests is failing and I'm no longer sure if it's really right to replace the clock.

Will push another commit to demo it

Reply:
See here: https://github.com/status-im/status-go/pull/2815/files#r953943414

Reply:
I see why, it's because we don't set that field on public chats, only on one to ones, which is a bit annoying. We can address in a separate issue I suppose